assets/src/components/propertiesPanel/status_run_block_tabs.tsx

```diff
@@ -49,13 +52,13 @@ const StatusRunBlockTabs = ({ statusContent }: Props) => (
       className="m-tabs__tab-panel"
       selectedClassName="m-tabs__tab-panel--selected"
     >
-      <h2>Run Content</h2>
+      {activeTripId && <MinischeduleRun activeTripId={activeTripId} />}
```

Comment:
This is likely a question for Ingrid, but if there is no active trip, do we want to display some message, or not show tabs in the first place or something?

Reply:
@ingridpierre ?